### PR TITLE
feat:[RP-27]-Replace text components with typography variant

### PR DESCRIPTION
**Pull request by dipSyxx**
# Title
Replace text components with typography variant

## Ticket Link
https://rozetka-home-project.atlassian.net/browse/RP-27

## Changes
!Added props to the typography component:
- marginBottom: mb-24, mb-16, mb-12, mb-8
- colorVariant: gray

!Added to typography styles:
- h3, h4, h6, subtitle1, subtitle2 and styles  for marginBottom and colorVariant

### CODE REVIEW

File: src/elements/Typography/index.tsx [hunk 1]
```diff
@@ -7,13 +7,21 @@ type TypographyProps = {
...
+  colorVariant?: 'gray'
 }
 
-export const Typography: FC<TypographyProps> = ({ children, component = 'h1', variant = 'h1' }) => {
+export const Typography: FC<TypographyProps> = ({
+  children,
+  component = 'h1',
+  variant = 'h1',
+  marginBottom = '',
+  colorVariant = '',
```
Comment (MaksPapirovnyk) [inline src/elements/Typography/index.tsx:19]: 👍

**Review by MaksPapirovnyk — CHANGES_REQUESTED** [review]
1. Please double check if your linter works
2. Take a look at how you use Typography, you don't need to wrap a string in curly brackets

File: src/elements/Typography/typographyStyles.module.sass [hunk 2]
```diff
@@ -4,10 +4,44 @@
...
   margin-bottom: 24px
 
 .h2
   font-size: 20px
   margin-bottom: 16px
+
+.h3
+  font-size: 24px
+  overflow: hidden
+  white-space: nowrap
```
[the remark this answers is not in the document]
Comment (dipSyxx) [reply src/elements/Typography/typographyStyles.module.sass:33]: yes, I also think it will be unnecessary :)

**Review by MaksPapirovnyk — APPROVED** [review]
LGTM

**Pull request merged**